pkg/sethandler/controller.go

```diff
@@ -284,9 +285,23 @@ func determineCid(podStatus v1.PodStatus, containerName string) string {
 	return ""
 }
 
+func trimContainerPrefix(contName string) (string, error) {
+	for _, prefix := range containerPrefixList{
+		if strings.HasPrefix(contName, prefix){
+			return strings.TrimPrefix(contName, prefix), nil
+		}
+	}
+	return "",  fmt.Errorf("container name does not contain predefined prefixes (%s) container name: %s",containerPrefixList, contName)
+}
+
 func containerIDInPodStatus(podStatus v1.PodStatus, containerDirName string) bool {
 	for _, containerStatus := range podStatus.ContainerStatuses {
-		trimmedCid := strings.TrimPrefix(containerStatus.ContainerID, "docker://")
+		trimmedCid, err := trimContainerPrefix(containerStatus.ContainerID)
+		if err != nil {
+			log.Printf("ERROR: %s\n", err)
```

Comment:
align the rror message to the one below this, mention it happened during trimming